src/commands/commerce/store/quickstart/setup.ts

```diff
@@ -116,13 +117,19 @@ export class StoreQuickstartSetup extends SfdxCommand {
         // Copy all example files
         FILE_COPY_ARGS.forEach((v) => modifyArgFlag(v.args, v.value, this.argv));
         await FilesCopy.run(addAllowedArgs(this.argv, FilesCopy), this.config);
+
         this.devHubUsername = (await this.org.getDevHubOrg()).getUsername();
         this.statusFileManager = new StatusFileManager(
             this.devHubUsername,
             this.org.getUsername(),
             this.flags['store-name'] as string
         );
         const storeType = StoreQuickstartSetup.getStoreType(this.org.getUsername(), this.flags, this.ux, this.logger);
+        const storeTemplate = this.getTemplateName();
+
+        // LWR uses DigitalExperienceBundle, B2B Aura uses ExperienceBundle.
+        this.isUsingDigitalExperienceBundle = storeType !== 'B2B' || storeTemplate !== 'B2B Commerce (Aura)';
```

Comment:
If you were on call and saw this line of code at 3 a.m. during a sev0, could you quickly understand what's happening and whether it was doing what the author intended? I know I'd struggle at 10 a.m. with coffee ;)
Please consider re-writing it to make it easy to follow and show off the intent.

A good rule of thumb is that if code needs a comment to explain it, the code should probably be reworked so it's more readable.

Reply:
Would you find this line more readable? 
` this.isUsingDigitalExperienceBundle = !(storeType === 'B2B' && storeTemplate === 'B2B Commerce (Aura)')`

Basically on here I am trying to set this flag to true when we are not creating a B2B Aura store. How would you approach this @dbarbato-sf ?